apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Controls/android.tsx

```diff
@@ -87,6 +87,15 @@ export const controlsPagesAndroid: INavPage[] = [
     url: '#/controls/android/popupmenu',
     isCategory: true,
     pages: [
+      {
+        title: 'Bottom Navigation',
```

Comment:
Typically (at least for now) we use the name of the control itself here, so BottomNavigation (same with TabBar)

Reply:
I think we've actually been doing the opposite so far for the mobile controls :). What's the goal for using the control name?

Reply:
Ah I see that now...ideally we should be doing the same for both, but that doesn't need to change right now. The main advantage I can see of using control names is to make it easier for developers to find docs on a particular control, but honestly the reason it's that way on the website now may be more "historical reasons" (it's always been that way) than anything else.

Reply:
Yeah that makes sense - I'll leave for now to be consistent in this section, but we can discuss holistically at some point.